### PR TITLE
Bhamelin/us125522 migrate attribute picker

### PR DESCRIPTION
**Pull request by BhamelinD2L**
Adds the attribute picker from https://github.com/BrightspaceUILabs/attribute-picker into this repo instead, and utilizes the `d2l-multi-select-list-item` as its list items. 

Readme, demo pages, lang terms, and tests updated to accommodate this component.
Adds a langterm into the list item X icon such that the delete icon, when displayed, makes it clear that item removal is possible.

### CODE REVIEW

File: demo/multi-select-list.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+						<span style="display: none;"></span>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 0"></d2l-labs-multi-select-list-item>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 1"></d2l-labs-multi-select-list-item>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 2"></d2l-labs-multi-select-list-item>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 3"></d2l-labs-multi-select-list-item>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 4"></d2l-labs-multi-select-list-item>
+						<d2l-labs-multi-select-list-item deletable="" text="List item 5"></d2l-labs-multi-select-list-item>
+					</d2l-labs-multi-select-list>
+				</template>
+			</d2l-demo-snippet>
```
[the remark this answers is not in the document]
Comment (BhamelinD2L) [reply demo/multi-select-list.html:59]: I believe it's to demo that you can insert other content into the lists without breaking the functionality. It was here when I found it 😅

File: attribute-picker.js [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+		if (this._attributeLimitReached()) {
+			this.dispatchEvent(new CustomEvent('d2l-attribute-limit-reached', {
+				bubbles: true,
+				composed: true,
+				detail: {
+					limit: this.limit
+				}
+			}));
+			return false;
+		}
```
[the remark this answers is not in the document]
Comment (BhamelinD2L) [reply attribute-picker.js:235]: Maybe, but I don't think changing it should be part of the scope of this story. A good time to address it is when the limits were being added. It should be easy to come back to it later though if we decide that's best

File: test/attribute-picker.test.js [hunk 3]
```diff
@@ -24,373 +24,425 @@ function evaluateListValues(array, elementList) {
...
-
-			//If we change the selection, ensure both the assigned list and assignable list are updated
-			el.attributeList = changedAttributeList;
-			await el.updateComplete;
-			attributeElements = el.shadowRoot.querySelectorAll('.d2l-attribute-picker-attribute');
-			expect(evaluateListValues(changedAttributeList, attributeElements)).to.equal(true);
-			dropdownElements = el.shadowRoot.querySelectorAll('.d2l-attribute-picker-li');
-			expect(evaluateListValues(expectedAssignableList, dropdownElements)).to.equal(true);
-
+	describe('d2l-labs-attribute-picker', () => {
```
Comment (ChrisLabattD2L) [inline test/attribute-picker.test.js:34]: Now that you wrapped the entire set of tests with `attribute-picker`, you don't need the `d2l-labs-attribute-picker` wrapper around the rest of the test.

Comment (BhamelinD2L) [reply test/attribute-picker.test.js:34]: My brain somehow completely missed this 👀  thanks

**Review by ChrisLabattD2L — APPROVED** [review]
Looks good! 😄

**Pull request merged**